MatrixSDK.xcodeproj/project.pbxproj

```diff
@@ -7237,6 +7243,7 @@
 				MACOSX_DEPLOYMENT_TARGET = 10.10;
 				MTL_ENABLE_DEBUG_INFO = YES;
 				ONLY_ACTIVE_ARCH = YES;
+				OTHER_SWIFT_FLAGS = "-D DEBUG";
```

Comment:
Note the addition of this flag, otherwise `DEBUG` macro in Swift is incompatible with the one in objective-c